app/uk/gov/hmrc/digitalservicestax/controllers/package.scala

```diff
@@ -23,7 +23,7 @@ package object controllers {
     def escapeHtml: String = HtmlFormat.escape(in).toString
   }
 
-  implicit class OrderedYesNo(l: List[String]) {
+  implicit class OrderedYesNo(val l: List[String]) extends AnyVal {
```

Comment:
Why is this needed?

Reply:
https://docs.scala-lang.org/overviews/core/value-classes.html#summary_of_limitations

Reply:
It's a performance optimisation, it's not strictly needed, but it makes perfect sense to have it in this type of scenario.

Reply:
Oh awesome, lgtm then.